index.js

```diff
@@ -370,7 +381,14 @@ class Raspi extends EventEmitter {
           break;
         case PWM_MODE:
         case SERVO_MODE:
-          pinInstance.peripheral = new PWM(normalizedPin);
+          if (this[pins][normalizedPin][isHardwarePwm] === true) {
```

Comment:
Style nit: we don't need to do the comparison with true, we can just let it be truthy. With the change to putting this property on the pin instance state object, it can be further simplified to:

```JavaScript
if (pinInstance.isHardwarePwm) {
```